### ISSUE TITLE
This problem occurred at the beginning of training....

### ISSUE THREAD

**Issue by yinminzzy** [hologerry/Attr2Font#22]
ValueError: Expected more than 1 spatial element when training, got input size torch.Size([32, 165, 1, 1])

I want to ask how to solve this problem,thanks

**Comment by Pesciolini** [comment]
I also encountered this problem, did you solve it and how did you solve it?

**Comment by yinminzzy** [comment]
yes....This seems to be due to the use of batch normalization, so H and W here must be numbers greater than 1.But I have forgotten where I modified it. I'm very sorry, you can debug it yourself

**Comment by HRHRHRHR666** [comment]
I also encountered this problem. Can you please provide a detailed solution?

**Comment by Pesciolini** [comment]
> yes....This seems to be due to the use of batch normalization, so H and W here must be numbers greater than 1.But I have forgotten where I modified it. I'm very sorry, you can debug it yourself

Thanks, use torch 1.4 & python 3.6 solve my problem

**Comment by Djs-Champion** [comment]
> ValueError: Expected more than 1 spatial element when training, got input size torch.Size([32, 165, 1, 1])
> 
> I want to ask how to solve this problem,thanks

Hello, have you resolved your issue?

**Comment by Djs-Champion** [comment]
> I also encountered this problem. Can you please provide a detailed solution?

Hello, have you resolved your issue?

**Comment by Djs-Champion** [comment]
> ValueError: Expected more than 1 spatial element when training, got input size torch.Size([32, 165, 1, 1])
> 
> I want to ask how to solve this problem,thanks

Hello, have you resolved your issue?
